nixos/modules/services/web-servers/nginx/tailscale-auth.nix

```diff
@@ -0,0 +1,173 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.nginx.tailscaleAuth;
+in
+{
+  options.services.nginx.tailscaleAuth = {
+    enable = mkEnableOption (lib.mdDoc "tailscale.nginx-auth");
```

Comment:
This should be a human readable name.

Reply:
Please refer to the discussion regarding the name above and/or provide me concrete suggestion or definition of "human readable" in this context :)